### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by ValterVB** [variar/klogg#261]
Details for the issue
--------------------

Search without regex

-------------------------
Crash id
d3aa2827-3bd3-4fea-8e92-a64210cb3554
-------------------------
Useful extra information
-------------------------
> Klogg version 20.12.0.810 (built from commit f473845) [built for x86_64-little_endian-llp64]
> running on Windows 10 Version 2009 (winnt/10.0.19042) [x86_64]
> and Qt 5.15.2

**Comment by variar** [comment]
Sentry issue KLOGG-3H
Thread 29544 Crashed:
```
0   KERNELBASE.dll                  0x7ffba25cd759      RaiseException
1   VCRUNTIME140.dll                0x7ffb9154621f      _CxxThrowException (throw.cpp:74)
2   Qt5Core.dll                     0x7ffb5e41b801      QUnhandledException::raise
3   Qt5Core.dll                     0x7ffb5e41dc05      QFutureInterfaceBase::waitForFinished
4   klogg_portable.exe              0x7ff7b35dae8d      LogFilteredDataWorker::search (logfiltereddataworker.cpp:252)
5   klogg_portable.exe              0x7ff7b35af4a4      LogFilteredData::runSearch (logfiltereddata.cpp:134)
6   klogg_portable.exe              0x7ff7b358e720      CrawlerWidget::replaceCurrentSearch (crawlerwidget.cpp:1094)
7   klogg_portable.exe              0x7ff7b358a134      CrawlerWidget::startNewSearch (crawlerwidget.cpp:355)
8   Qt5Core.dll                     0x7ffb5e5e1d48      QObject::qt_static_metacall
9   Qt5Widgets.dll                  0x7ffb6e29eca6      QAbstractButton::clicked
10  Qt5Widgets.dll                  0x7ffb6e29eaa9      QAbstractButton::click
11  Qt5Core.dll                     0x7ffb5e5e1d48      QObject::qt_static_metacall
12  Qt5Widgets.dll                  0x7ffb6e2e766d      QLineEdit::qt_static_metacall
13  Qt5Core.dll                     0x7ffb5e5e1da6      QObject::qt_static_metacall
14  Qt5Widgets.dll                  0x7ffb6e2f2221      QWidgetLineControl::processKeyEvent
15  Qt5Widgets.dll                  0x7ffb6e2ead04      QLineEdit::keyPressEvent
16  Qt5Widgets.dll                  0x7ffb6e1f7a8a      QWidget::event
17  Qt5Widgets.dll                  0x7ffb6e2ea0de      QLineEdit::event
18  Qt5Widgets.dll                  0x7ffb6e2b46c8      QComboBox::keyPressEvent
19  Qt5Widgets.dll                  0x7ffb6e1f7a8a      QWidget::event
```

Could be some data race.

**Comment by variar** [comment]
Could be fixed in 21.04.0.844. Please open another issue with new crash id if crashes repeat.
